spec/javascripts/health-care/components/full_name.spec.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-addons-test-utils';
+import DateInput from '../../../../_health-care/_js/_components/full_name';
+import _ from 'lodash';
+
+describe('<FullName>', () => {
+  let component = null;
+
+  beforeEach(() => {
+    component = ReactTestUtils.renderIntoDocument(
+      <FullName name={{first:"William", last:"Shakespeare"}} onUserInput={(update) => {}}/>
+    );
+    assert.ok(component, "Cannot even render component");
+  });
+
+  it('has sane looking features', () => {
+    let inputs = ReactTestUtils.scryRenderedDOMComponentsWithTag(component, 'input');
+    expect(inputs).to.have.length(3);
```

Comment:
This test is failing right now locally because it says it can't find variable `FullName`. I am trying to debug this now, but wanted to push up the code in the meantime!